test/testFunctions.js

```diff
@@ -48,6 +48,29 @@
     }
   }
 
+  function assertTransform (containerStyle, targetStyle, expectedTransform) {
+    var container = document.createElement('div');
+    document.body.appendChild(container);
```

Comment:
Perhaps move down to after 'container.appendChild(target);', so we append a final container subtree to body.